guides/common/modules/proc_installing-packages-on-a-host.adoc

```diff
@@ -7,7 +7,7 @@ The list of packages available for installation depends on the Content View and
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the host you want to install packages on.
 . On the *Content* tab, select the *Packages* tab.
-. On the vertical ellipsis icon next to the upgrade button, click *Install Packages*.
+. On the vertical ellipsis icon at the top, click *Install Packages*.
```

Comment:
We should avoid this kind of navigation as per the IBM Style Guide.

Reply:
Addressed :+1: